pages/snapshots/[id].js

```diff
@@ -157,8 +158,12 @@ const SnapshotSummary = ({ resources, initialSnapshot, token }) => {
 
           <div data-testid="notes-summary">
             <br></br>
-            <button className="govuk-button" id="print">Print this page</button><br></br>
-            <button className="govuk-button" id="shared-plan">Continue</button>
+            <a
+              href={`${INH_URL}/help-requests/complete/${customerId}`}
```

Comment:
The way this `customerId` is extracted from the URL looks a little fragile, but I think that is worth the tradeoff to get this working?

Reply:
@kjdchapman do you think I should get rid of the "inh-" prefix ? I am not sure what do you mean

Reply:
I have removed the prefix